data/json/mapgen/lake_buildings/freshwater_research_station.json

```diff
@@ -221,6 +221,7 @@
       ],
       "palettes": [ "underwater_structure" ],
       "place_loot": [
+        { "item": "shark_suit_faraday", "x": [ 9, 10 ], "y": 12, "chance": 60 },
```

Comment:
Why the faraday one, and not the normal one? I'm not an underwater researcher, but I don't imagine that they have these.

Reply:
good point.  I'll adjust it in a new PR.